src/macaron/__main__.py

```diff
@@ -242,6 +242,15 @@ def main() -> None:
         help=("The path to the Jinja2 html template (please make sure to use .html or .j2 extensions)."),
     )
 
+    single_analyze_parser.add_argument(
```

Comment:
Are we still using this cmd arg or should we remove it?

Reply:
It should have been removed.